src/payment/strategies/bluesnapv2/bluesnapv2-payment-options.ts

```diff
@@ -0,0 +1,29 @@
+
+/**
+ * A set of options that are required to initialize the BlueSnap V2 payment
+ * method.
+ *
+ * The payment step is done through a web page via an iframe provided by the
+ * strategy.
+ */
+export interface BlueSnapV2PaymentInitializeOptions {
+    /**
+     * A callback that gets called when the iframe is ready to be added to the
+     * current page. It is responsible for determining where the iframe should
+     * be inserted in the DOM.
+     *
+     * @param error - Any error raised during the payment process;
+     * undefined if there is none.
+     * @param iframe - The iframe element containing the payment web page
+     * provided by the strategy.
+     * @param cancel - A function, when called, will cancel the payment
+     * process and remove the iframe.
+     */
+    addFrame(error: Error | undefined, iframe: HTMLIFrameElement, cancel: () => void): void;
```

Comment:
since these are callbacks, maybe it's better naming it something like `onLoad` and `onCancel` ? This way would be consistent with other payment methods.

https://github.com/bigcommerce/checkout-sdk-js/blob/a845e1c8062a51cb1df962b3a9d67d91b8cc8a11/src/payment/strategies/chasepay/chasepay-initialize-options.ts#L18

Reply:
I took as reference 3DS from Braintree but, I agree with you.